README.md

```diff
@@ -116,10 +116,6 @@ function streamStock(ticker) {
   });
 }
 
-function finish(stopMsg) {
-  socket.send(JSON.stringify(stopMsg));
-}
-
```

Comment:
Nice catch. I think the intention was to put `finish` up above on L106, instead of `socket.send(JSON.stringify(stopMsg))`. But this PR (keeping this function's contents inlined up above instead of using it) is probably simpler.